nuget/chocolatey/tools/chocolateysetup.psm1

```diff
@@ -30,13 +30,56 @@ param (
   }
 }
 
+function Remove-ShimWithAuthenticodeSignature {
+  param (
+    [string] $filePath
+  )
+  if (!(Test-Path $filePath)) {
+    return
+  }

```

Comment:
@AdmiringWorm the alignment doesn't look quite right here.  Can you push this over to line up with if statement below?

Reply:
This indentation was actually correct, however, the remaining of the function had the wrong indentation compared to the rest of the file.

I have fixed that up.